roborazzi-compose-preview-scanner-support/build.gradle

```diff
@@ -0,0 +1,28 @@
+plugins {
+  id 'com.android.library'
+  id 'org.jetbrains.kotlin.android'
+}
+if (System.getenv("INTEGRATION_TEST") != "true") {
+  pluginManager.apply("com.vanniktech.maven.publish")
+}
+repositories {
+  google()
+  mavenCentral()
+  maven { url = uri("https://jitpack.io") }
+}
+
+android.buildFeatures.compose = true
+
+// TODO: Use build-logic
+apply from: rootProject.file('gradle/android.gradle')
+
+android.namespace 'com.github.takahirom.roborazzi.preview.support'
+
+dependencies {
+  implementation project(":roborazzi")
+  implementation project(":roborazzi-compose")
+
+  compileOnly libs.androidx.compose.runtime
+  compileOnly libs.composable.preview.scanner
+  compileOnly libs.robolectric
```

Comment:
It could be a point of discussion, but the current approach of depending on other libraries is using compileOnly in Roborazzi to prevent making breaking changes.